resource/ui/ts/models/stats.ts

```diff
@@ -22,5 +22,61 @@ module Models {
             sys_count: number;
             last_update_nanos: number;
         }
+
+        // This function was adapted from
+        // https://stackoverflow.com/questions/10420352/converting-file-size-in-bytes-to-human-readable
+        export function FormatBytes(bytes: number): string {
+            var thresh = 1024;
+            if (Math.abs(bytes) < thresh) {
+                return bytes + ' B';
+            }
+            var units = ['KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB'];
```

Comment:
nit: you can simplify this function by prepending `'B'` to this array (and making it a constant) and indexing into it by `log2(bytes)/10`.

[log2](https://github.com/Microsoft/TypeScript/blob/master/src/lib/es6.d.ts#L541